packages/worker/src/sdk/tenants/tenants.ts

```diff
@@ -74,3 +74,10 @@ async function removeTenantUsers(tenantId: string) {
     throw err
   }
 }
+
+export async function tenantInfo(tenantId: string) {
+  const globalDbName = tenancy.getGlobalDBName(tenantId)
+  return {
+    exists: await dbCore.dbExists(globalDbName),
```

Comment:
This might give us a false positive. e.g. in qa, because of the db auto creation we end up with a db `budicloud` 

![Screenshot 2023-03-30 at 10 31 56](https://user-images.githubusercontent.com/8755148/228794113-d419da19-79e8-45e1-b55d-7dcc1d18addd.png)

Reply:
nevermind, just seen the follow up pr 👍